app/pb_kits/playbook/pb_user_badge/_user_badge.jsx

```diff
@@ -4,18 +4,18 @@ import React from 'react'
 type UserBadgeProps = {
   className?: String,
   id?: String,
-  badge?: 'million-dollar' | 'veteran' ,
+  badge: React.ReactNode,
```

Comment:
We might even try `'million-dollar' | 'veteran' | React.ReactNode` which will better follow the proposal in the comments..